web/Services/AzureStorage.cs

```diff
@@ -115,15 +115,21 @@ public async Task DeleteFile(string varType, int id, string blobName)
             }
         }
         
-        public bool IsImage(string fileName)
+        public (bool, string?) IsImage(string fileName)
         {
             string fileType = SplitFileName(fileName).type;
             List<string> imageExtensions = new List<string> { "jpg", "jpeg", "png", "bmp", "gif" };
             if (imageExtensions.Contains(fileType.ToLower()))
             {
-                return true;
+                return (true, null);
             }
-            return false;
+
+            string imageExtString = "";
+            foreach (string imageExt in imageExtensions)
+            {
+                imageExtString += imageExt + ", ";
+            }
+            return (false, imageExtString.Substring(0, imageExtString.Length - 2));

```

Comment:
This is `String.Join` <https://docs.microsoft.com/en-us/dotnet/api/system.string.join?view=net-6.0#system-string-join(system-string-system-collections-generic-ienumerable((system-string)))>